packages/klevu-ui/src/components/klevu-button/klevu-button.css

```diff
@@ -12,6 +12,9 @@
   --padding: var(--klevu-button-padding, calc(var(--klevu-size-m) - (var(--klevu-body-s-size) / 2)));
   --font-size: var(--klevu-button-font-size, var(--klevu-body-s-size));
   --text-color: var(--klevu-button-text-color, var(--klevu-color-neutral-1));
+  --icon-size: var(--klevu-icon-size, 1.5em);
```

Comment:
rename to --klevu-button-icon-size, same for the other 3

Reply:
name changed